templates/base/generators/stores/Context.jsx.hbs

```diff
@@ -1,18 +0,0 @@
-/**
```

Comment:
Why are you deleting this one? This was a template to generate a new store if someone needs to. If we are deleting it, this should be removed from Plop options

Reply:
Since we are replacing it with smart-context, I deleted those files and added smart-context to plop generator. If we are removing it from plop options, then we don't need those files in the generator directory?